pkg/utils/securitycontext.go

```diff
@@ -0,0 +1,211 @@
+package utils
+
+import (
+	"encoding/json"
+	"errors"
+	v1 "k8s.io/api/core/v1"
+	"keptn-contrib/job-executor-service/pkg/config"
+	"log"
+	"os"
+)
+
+// jobSecurityContextFilePath describes the path of the security config file that is defined in the deployment.yaml
+const jobSecurityContextFilePath = "/config/job-defaultSecurityContext.json"
+
+// podSecurityContextFilePath describes the path of the pod security config file that is defined in the deployment.yaml
+const podSecurityContextFilePath = "/config/job-podSecurityContext.json"
+
+// ReadDefaultJobSecurityContext reads the JSON file defined in jobSecurityContextFilePath and parses the output
+// into a valid v1.SecurityContext data structure
+func ReadDefaultJobSecurityContext() (*v1.SecurityContext, error) {
+	jobSecurityContextBytes, err := os.ReadFile(jobSecurityContextFilePath)
+	if err != nil {
+		return nil, err
+	}
+
+	var defaultJobSecurityContext v1.SecurityContext
+	err = json.Unmarshal(jobSecurityContextBytes, &defaultJobSecurityContext)
+	if err != nil {
+		return nil, err
```

Comment:
This is a problem with the content of the file itself (we managed to read), so
```suggestion
		return nil, fmt.Errorf("invalid security context configuration format: %w", err)
```